src/components/cluster/detail/Versions.js

```diff
@@ -25,17 +34,17 @@ function Versions({
           <RefreshableLabel dataItems={[releaseVersion]}>
             <>
               <Dot style={{ paddingRight: 0 }} />
-              <i className='fa fa-version-tag' />
-              <span
+              <ReleaseDetail
                 className='pointer'
                 onClick={() => releaseDetailsModal.current.show()}
               >
+                <i className='fa fa-version-tag' />
                 {releaseVersion ? releaseVersion : 'n/a'}
-              </span>
+              </ReleaseDetail>
             </>
           </RefreshableLabel>
         </span>
-        <span
+        <ReleaseDetail
           className='pointer'
```

Comment:
We can remove this class, this was what we were actually using for the underline + pointer styles on hover. Think I lost the css when I crated this component